index.html

```diff
@@ -9,7 +9,7 @@
 <body>
   <!-- Navigation start -->
   <nav>
-    <div class="brandNameBox"><p class="brandName">Gourmet au Catering</p></div>
+    <div class="brandNameBox"><p class="brandName"><a href="#">Gourmet au Catering</a></p></div>
```

Comment:
No need of multiple block elements like div, p etc. Include any one or use <a> alone and style to implement the design.

Reply:
Multiple block elements removed.